model-optimizer/extensions/front/onnx/aten_ext.py

```diff
@@ -26,7 +26,6 @@ class ATenFrontExtractor(FrontExtractorOp):
     def extract(cls, node):
         mode = onnx_attr(node, 'mode', 'i', default=1)
         operator = onnx_attr(node, 'operator', 's').decode()
-        scale_grad_by_freq = onnx_attr(node, 'scale_grad_by_freq', 'i', default=0)
```

Comment:
Why are we removing this attribute extraction?

Reply:
This attribute was added by mistake. It is used only during training.